### DISCUSSION TITLE
Table filter with backend API

### DISCUSSION THREAD

**Discussion by MartinKoeditz** [primefaces/community#1218] [PrimeNG]
Hi,

I'm currently working on my new PrimeNG project. Everything's fine so far. But I have one point that's not clear to me. I have a table filter row and an global search field. When filtering data I want to send the filter request to my API backend and show the results in the table. It's already working for the filter row, but I am unable to do it including the global search, since it wants to be configured with the columns.

Is there an example or something I can use as a start point? I Hope, someone can help me.

**Comment by cetincakiroglu** [comment]
Hi,

This example would help https://primeng.org/table#customers

**Reply by MartinKoeditz** [reply]
Sorry, it does not. I already seen the filter examples. But they work internally and don't send data to backend. Or do I miss something?

**Reply by MartinKoeditz** [reply]
Well, I found a solution. Thanks to YT video:
https://www.youtube.com/watch?v=4qaIJehpYaw&ab_channel=LearnfromScratch
